pkg/cmd/provisioning/aws/constants.go

```diff
@@ -1,33 +0,0 @@
-package aws
```

Comment:
for git history purposes, can this file show up as a rename to pkg/cmd/provisioning/constants.go ?

Reply:
I tried but as there are changes in that file git does not show it as rename

Reply:
wierd b/c it seems to have worked for pkg/cmd/provisioning/aws/create_key_pair.go → pkg/cmd/provisioning/create_key_pair.go , oh well

Reply:
true, no clue what is happening for constants.go